### ISSUE TITLE
JSDocs validate description

### ISSUE THREAD

**Issue by fsmanuel** [eslint/eslint#2449]
Some JSDoc standards require to start a JSDoc description with a capital letter and end it with a dot.

That should be easy to implement but I'm unsure about the option name and value. A regex would be a good start:

```json
{
  "rules": {
    "valid-jsdoc": [2, {
      "validateDescription": {"match": "/^[A-Z]*.$/"}
    }]
  }
}
```

```js
// valid-jsdoc.js

if (validateDescription) {
  const regex = new RegExp(validateDescription.match);
  
  if (!regex.exec(jsdoc.description)) {
    context.report(jsdocNode, "JSDoc description must start with a capital letter and end with a dot.");
  }
}
```

What do you think? With enough feedback I'll open a PR.

**Comment by yfr** [comment]
Uh just what i looked for!

:+1:

**Comment by nzakas** [comment]
We need to work on the option name, but seems useful.

**Comment by mathieumg** [comment]
:+1:

**Comment by fsmanuel** [comment]
what about:

```json
{
  "rules": {
    "valid-jsdoc": [2, {
      "matchDescription": "/^[A-Z]*.$/"
    }]
  }
}
```

or to be more precise `matchDocBlockDescription` which ever you prefer.

**Comment by nzakas** [comment]
That's an interesting point, would this pattern apply to all descriptions? What about `@param` descriptions? Or `@fileoverview` descriptions?

**Comment by fsmanuel** [comment]
@nzakas I guess other descriptions should have different RegEx patterns.

**Comment by gyandeeps** [comment]
Do we need different description option or only single option which applies to all or should we just check the description only?

@eslint/eslint-team

**Comment by nzakas** [comment]
Let's start by just checking description for the block.

**Comment by fsmanuel** [comment]
@gyandeeps :+1:
